privaterelay/locales/en-us/app.ftl

```diff
@@ -0,0 +1,239 @@
+
+# Variables:
+#   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
+#   $attrs (string) - specific attributes added to external links
+how-it-works-step-1-copy-html = <a href="{ $url }" { $attrs }>Download the { -brand-name-relay } extension for { -brand-name-firefox }</a>. 
+    Select the icon that appears on your { -brand-name-firefox } toolbar to access the sign-in page. 
+    Sign in with your { -brand-name-firefox-account } to get started.
+how-it-works-step-2-headline = Create a new alias
+how-it-works-step-2-copy = As you browse, the { -brand-name-relay } icon will appear in form fields where sites ask for your email address. 
+    Select it to generate a new, random address that ends in @relay.firefox.com.
+    { -brand-name-relay } will forward messages to the primary email address associated with your account.
+how-it-works-step-3-headline = Manage your account
+how-it-works-step-3-copy = Sign in to the { -brand-name-relay } website to keep track of the aliases you’ve created.
+    If you find that one receives spam or unwanted messages, you can block all messages or even delete the alias, 
+    right from the management page.
+hero-image-copy-trust = Can you even trust this company with your personal email?
+hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for every new account …
+hero-image-copy-protect-html = …so you can <strong>protect your real email</strong> from tracking and spam.
+hero-image-copy-control-html = Now <em>you’re in control</em> of what’s arriving to your inbox!
+
+## FAQ Page
+
+faq-headline = Frequently Asked Questions
+faq-question-1-question = What about spam?
+faq-question-1-answer-a = While { -brand-name-relay } does not filter for spam, our email partner Amazon SES does block spam and malware. If { -brand-name-relay } forwards messages you don’t want, you can update your { -brand-name-relay } settings to block messages from the alias forwarding them.  
+
+# Variables:
+#   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
+#   $attrs (string) - specific attributes added to external links
+faq-question-1-answer-b-html = If you see a broader problem of unwanted email from all of your aliases, please <a href="{ $url }" { $attrs }>report this to us</a> so we can consider adjusting the SES spam thresholds for this service. If you report these as spam, your email provider will see { -brand-name-relay } as the source of spam, not the original sender.
+faq-question-2-question = Why won’t a site accept my { -brand-name-relay } alias?
+
+# Variables:
+#   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
+#   $attrs (string) - specific attributes added to external links
+faq-question-2-answer-html = Some sites may not accept an email address that includes a subdomain (ie, the “relay” portion of @relay.firefox.com) and others have stopped accepting all addresses except those from Gmail, Hotmail, or Yahoo accounts. 
```

Comment:
ie or i.e.?

Reply:
"i.e." is the correct usage from what I can gather.

Reply:
(Updated to i.e.)